project/AppDependencies.scala

```diff
@@ -0,0 +1,32 @@
+import play.core.PlayVersion
+import sbt._
+
+object AppDependencies {
+  private lazy val silencerVersion                   = "1.7.9"
```

Comment:
I can see this file is not formatted with scalafmt please do

Reply:
done